### PR TITLE
Add multi cassette on dashboard

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Cassette 3 and 4 missing on dashboard](https://trello.com/c/VUOiirBI/1047-cassette-3-and-4-missing-on-dashboard)

File: new-lamassu-admin/src/pages/Machines/MachineComponents/Cassettes/Cassettes.js [hunk 1]
```diff
@@ -1,18 +1,26 @@
+/* eslint-disable no-unused-vars */
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Machines/MachineComponents/Cassettes/Cassettes.js]: Enable eslint

File: new-lamassu-admin/src/pages/Machines/MachineComponents/Cassettes/Cassettes.js [hunk 2]
```diff
@@ -87,87 +106,44 @@ const CashCassettes = ({ machine, config, refetchData }) => {
...
+          decimalPlaces: 0,
+          width: widthsByNumberOfCassettes[numberOfCassettes].cassetteGraph,
+          inputClassName: classes.cashbox
+        }
+      })
+      return R.add(1, it)
+    },
+    1
+  )
+
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Machines/MachineComponents/Cassettes/Cassettes.js:145]: Nice one!

**Review by josepfo — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Machines/MachineComponents/Cassettes/Cassettes.js (hunk 1 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Machines/MachineComponents/Cassettes/Cassettes.js]: Ah crap, development leftover. Nice catch

**Review by josepfo — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**